tests/rt.sh

```diff
@@ -417,9 +417,9 @@ if [[ $SINGLE_NAME != '' ]]; then
 fi
 
 if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = cheyenne.* ]]; then
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20200805/${COMPILER^^}}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20200806/${COMPILER^^}}
 else
-  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20200805}
+  RTPWD=${RTPWD:-$DISKNM/NEMSfv3gfs/develop-20200806}
```

Comment:
Sorry, I may miss this part, may I ask why the baseline results are changed? Is it due to the fv3_stochy test?

Reply:
`fv3_ccpp_ca` test results changed because of the bugfix in the cellular automata code. See https://github.com/noaa-psd/stochastic_physics/pull/27 and https://github.com/noaa-psd/stochastic_physics/issues/24.

Reply:
Thanks